.github/workflows/docs.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     - uses: ./.github/workflows/setup-with-retry
     - name: Build openpilot
       run: |
-        ${{ env.RUN }} "scons -j$(nproc)"
+        ${{ env.RUN }} "scons -j$(nproc) --warn=all"
```

Comment:
can we enable `--warn=all` as the default setting in the SConstruct instead?

Reply:
Changed